service/metrics/Dockerfile

```diff
@@ -0,0 +1,87 @@
+# This dockerfile exemplifies how a unitxt metrics service may be containerized
+
+FROM registry.access.redhat.com/ubi8/ubi:latest
+
+# Disable Red Hat Subscripstion
+RUN sed -i 's/1/0/g' /etc/yum/pluginconf.d/subscription-manager.conf
+
+# Set up conda env var
+ENV CONDA_HOME=/opt/conda \
+    PATH=/opt/conda/bin:/usr/local/nvidia/bin:$PATH \
+    CONFIG_DIR=/root/config_dir \
+    BUILD_DIR=/tmp/wml_report \
```

Comment:
Why WML?

Reply:
fixed, changed to `BUILD_DIR=/tmp/unitxt_metric_service`